index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio Setup</title>
+    <link rel="stylesheet" href="style.css">
+    <!-- font awesome icons -->
+    <script src="https://kit.fontawesome.com/25935046d9.js" crossorigin="anonymous"></script>  
+</head>
+<body>
+    <header class="header">
+        <a href="index.html" class="my_logo">Abeni-Logo</a>
```

Comment:
- [ ] This logo on line 14 seems a little off, even after importing the correct font-weight 700.  It seems the `font-size` is incorrect and the alignment too.  You could use `flex` to resolve the alignment issue.  

Look, you could try this:

![image](https://user-images.githubusercontent.com/57421823/148647711-f70c7b45-b20d-4bee-bbbd-cdc5040a0226.png)

![image](https://user-images.githubusercontent.com/57421823/148647717-c0fbb2a4-74e6-4002-beb0-1de4ffa7125c.png)

☝️ This is just an example of how it could be done, (after importing the correct font-weight 700 mentioned in another comment).

That way, the logo will end up perfectly aligned and with the correct size, like in the design 💪

Reply:
Thanks a lot for giving an example to look for.